custom_components/tahoma/switch.py

```diff
@@ -64,19 +64,23 @@ async def async_turn_on(self, **_):
             await self.async_execute_command(COMMAND_ON)
 
         elif self.has_command(COMMAND_RING_WITH_SINGLE_SIMPLE_SEQUENCE):
-            # Values taken from iosiren.js (tahomalink.com). Parameter usage is currently unknown.
             await self.async_execute_command(
-                COMMAND_RING_WITH_SINGLE_SIMPLE_SEQUENCE,
-                120000,
-                75,
-                2,
+                COMMAND_RING_WITH_SINGLE_SIMPLE_SEQUENCE,  # https://www.tahomalink.com/enduser-mobile-web/steer-html5-client/vendor/somfy/io/siren/const.js
+                2 * 60 * 1000,  # 2 minutes
+                75,  # 90 seconds bip, 30 seconds silence
```

Comment:
And so 90 and 30 give 75?

Reply:
do you actually doubt Somfy's wisdom??

Reply:
![](https://media.giphy.com/media/XyOrJljDNBEpa/giphy.gif)